app/components/Checkbox.tsx

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react'
+import * as CheckboxPrimitive from '@radix-ui/react-checkbox'
+import cx from 'classnames'
+
+import CheckIcon from '../icons/Check'
+
+type CheckboxProps = {
+  /**
+   * used for the label's `for` attribute
+   */
+  id: string,
+  label: string,
+  /**
+   * @default false
+   */
+  checked: boolean,
+  disabled?: boolean,
+  onChange: (checked: boolean) => void
+}
+
+const ControlledCheckbox = ({ id, label, checked, disabled, onChange }: CheckboxProps) => {
+  return (
+    <div className="flex justify-center items-center space-x-2">
+      <CheckboxPrimitive.Root
+        onCheckedChange={onChange}
+        checked={checked}
+        disabled={disabled}
+        id={id}
+        className={cx('w-4 h-4 flex items-center mx-[1px] border border-gray-300 rounded focus:outline-none focus:border-indigo-500 focus:border-2 focus:w-[18px] focus:h-[18px] focus:mx-0', {
```

Comment:
You should use consistent colors. Look at the borders and the button color. Its different and its weird...

Reply:
fixed